syncd/VirtualOidTranslator.cpp

```diff
@@ -147,6 +147,79 @@ sai_object_id_t VirtualOidTranslator::translateRidToVid(
     return vid;
 }
 
+void VirtualOidTranslator::translateRidToVid(
```

Comment:
please analyse funcion in line 60, this method here dont translare removed objects

Reply:
I don't pass removedObjects = true in the code, would requires coverage by some code that actually uses it